### PR TITLE
[CLOUDCRAFT-532] How does Cloudcraft connect to my Azure account?

### PR DESCRIPTION
**Pull request by jamesponddotco**
### What does this PR do? What is the motivation?
Add an article informating customers how we connect to their Azure accounts, just like we did for AWS.

### Merge instructions
Please wait until the Cloudcraft team has reviewed the content before going ahead with the review and merging of the article.

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/cloudcraft/faq/how-cloudcraft-connects-to-azure.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Cloudcraft connects to your Azure account using the IAM role created in your Azure subscription. This role allows Cloudcraft to communicate with Microsoft's public API and query the metadata that describes your infrastructure. The discovery process is read-only and does not require opening up your network or connecting directly to your infrastructure.
+
+Here's how the process works:
+
+1. Cloudcraft makes several API requests to [every supported Azure component][1].
+2. The results are correlated to reverse engineer your infrastructure into an architecture diagram.
+3. This process is repeated every time you run a scan to ensure your diagram is up-to-date.
+
+The only exception to the read-only rule is when an AKS Cluster is discovered. In this case, you have the option to connect Cloudcraft directly to the cluster via Kubernete's native APIs for complete workload and pod discovery. For more information, see [Connect an Azure AKS Cluster with Cloudcraft][2].
```
Comment (estherk15) [inline content/en/cloudcraft/faq/how-cloudcraft-connects-to-azure.md:14]:
```suggestion
The only exception to the read-only rule is when an AKS Cluster is discovered. In this case, you have the option to connect Cloudcraft directly to the cluster through Kubernetes' native APIs for complete workload and pod discovery. For more information, see [Connect an Azure AKS Cluster with Cloudcraft][2].
```

**Review by estherk15 — APPROVED** [review]

**Comment by jamesponddotco** [comment]
The first paragraph is quite long. Maybe I should break `The discovery process is read-only and does not require opening up your network or connecting directly to your infrastructure.` into its own paragraph.

What do you think, @estherk15?

**Comment by estherk15** [comment]
Thanks for asking @jamesponddotco! I don't really mind the length, since the overall page is quite concise. I think it gets to the point and answers the question. You could add a Header to break up the paragraphs, also leaving it seems good to me too.

**Comment by jamesponddotco** [comment]
@estherk15 Sounds good. We're ready to merge, then :)

**Pull request merged**